### PR TITLE
First project

### PR DESCRIPTION
**Pull request by haftamudesta**
this is my first project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
-# first-project
\ No newline at end of file
+# Project Name:hello Microverse
+This is my first project at Microverse.
+## Built With
+.HTML
+.CSS
+## Autor
```
Comment (TedLivist) [inline README.md:6]: - [ ] You have a typo error here, kindly fix it.

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+.Github: Destahaftamu(ttps://github.com/haftamudesta)
+Linkedin:haftamu2007@gmail.com(https://www.linkedin.com/in/haftamu-desta-795791a1/)
+.Tiwtter:DestaHaftamu(https://twitter.com/DestaHftamu?t=NQ4ovkdWbsfsjh62NFEXFg&s=09)
```
Comment (TedLivist) [inline README.md:9]: - [ ] These links are disorganized and some are incorrect. Kindly fix them. You can see suggestions below:
  ```
  [Github](https://github.com/haftamudesta)
  [LinkedIn](https://www.linkedin.com/in/haftamu-desta-795791a1)
  [Twitter](https://twitter.com/DestaHftamu)
  ```

File: README.md [hunk 3]
```diff
@@ -1 +1,17 @@
...
+
+## Contributing
+Contributions, issues, and feature requests are welcome!
+
+##Acknowledgments
+Microverse
+License
+This project is Apache License licensed.
```
Comment (TedLivist) [inline README.md:17]: - [ ] I would recommend you add other missing sections, like the **Getting Started** section describing how to set up and run the app locally from the Microverse template [here](https://github.com/microverseinc/readme-template).

Comment (TedLivist) [inline README.md:17]: - [ ] Kindly format your documentation to be neater and add the other sections as we have them in the template:

  hafta-contrib

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @haftamudesta,

Good job so far!
✔️ `test.md` file is ignored
✔️ Hello Microverse is displayed
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1 +1,23 @@
...
+# Project Name:hello Microverse
+This is my first project at Microverse.
+## Built With
+.HTML
+.CSS
+## Getting Started
+to implement and run this project you should have the following prerequisites  
+## Prerequisites
+Text editor
+browser like crome
```
Comment (ShoiraTa) [inline README.md:10]: - [ ] OPTIONAL: Please kindly fix the type 'crome'

File: README.md [hunk 5]
```diff
@@ -1 +1,23 @@
-# first-project
\ No newline at end of file
+# Project Name:hello Microverse
+This is my first project at Microverse.
+## Built With
+.HTML
+.CSS
+## Getting Started
+to implement and run this project you should have the following prerequisites
```
Comment (ShoiraTa) [inline README.md:7]: - [ ] OPTIONAL: You can consider adding more steps on how others can clone and get started with your project 😉

File: README.md [hunk 6]
```diff
@@ -1 +1,23 @@
...
+browser like crome
+## Author
+ [Githab](https://github.com/settings/profile)
+[linkedin](https://www.linkedin.com/in/haftamu-desta-795791a1/)
+[tiwtter](https://twitter.com/DestaHftamu?t=NQ4ovkdWbsfsjh62NFEXFg&s=09)
+
+## Contributing
+Contributions, issues, and feature requests are welcome!
+## Show your support
+give tumps if you like this project.
```
Comment (ShoiraTa) [inline README.md:19]: - [ ] OPTIONAL: Please kindly fix the typo in 'tumps'

**Review by ShoiraTa — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @haftamudesta ,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

parks-and-rec-great-job-gif

## Optional suggestions

- [ ] Please consider giving a descriptive title for Pull Request. Currently it is 'First project'
A good PR title would be `Set up a "Hello Microverse" project`

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

***
Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**